external/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/OracleIntegrationSuite.scala

```diff
@@ -69,6 +63,7 @@ class OracleIntegrationSuite extends DockerJDBCIntegrationSuite with SharedSpark
   }
 
   override def dataPreparation(conn: Connection): Unit = {
+    conn.setAutoCommit(false)
```

Comment:
In 18.4.0 Express Edition auto commit is enabled by default.

Reply:
Could you leave the comment in the code side?

Reply:
I've taken over this into the separate PR.